docs/topics/mta-rn-new-features-1.adoc

```diff
@@ -8,6 +8,6 @@
 
 This section describes the new features and improvements of the {ProductName} ({ProductShortName}) {ProductVersion}.
 
-.Support for non-administrators to run {ProductShortName}
-For security reasons, the RWX permission set of {ProductShortName} 6.0.0 was configured to allow only administrators to run {ProductShortName}. In Version {ProductVersion}  you can now allow non-administrators to use {ProductShortName} by setting the value of a new parameter, `rwx_supported`, to `false` in the the {ProductShortName} Operator or the {ocp-first}  console.
+.{ProductShortName} no longer requires RWX permissions to store data in PVs
+Previously, {ProductShortName} required RWX permissions to store data in persistent volumes (PVs). In Version {ProductVersion} you can set a new parameter, `rwx_supported`, to `false`, which allows {ProductShortName} to store data in PVs with other permission settings.
```

Comment:
Please use the below text instead. It is based on the enhancement description in MTA-44:
MTA no longer requires support for RWX volumes. 

Previously MTA required support for RWX volumes, and in version 6.0.1 this became optional.
The Tackle CR includes a new configuration option rwx_supported (bool) default: true. When RWX is supported, the MTA operator will create a cache volume as RWX. This volume is used by maven as durable local m2 repository that is shared across tasks. When not supported, the maven m2 cache will be local with each task. The user always has the ability to specify an actual maven cache.